src/components/AddBook.js

```diff
@@ -0,0 +1,18 @@
+const AddBook = () => (
+  <section id="addBook">
+    <h2>ADD NEW BOOK</h2>
+    <form>
+      <br />
+      <input type="text" id="addBook" name="addBook" placeholder="new book" />
+      <select name="category" id="category">
+        <option value="Drama" selected>Drama</option>
+        <option value="Economy" selected>Economy</option>
+        <option value="Science Fiction">Science Fiction</option>
+        <option value="Action">Action</option>
+      </select>
+      <input type="submit" value="Submit" />
```

Comment:
- [ ] This component should render a form with inputs for both the title and author of the book. You're currently having only the title input. Add the author input to your form.